mode_blueprints/verify_run_status.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import sys
+import requests
+from requests.auth import HTTPBasicAuth
+import shipyard_utils as shipyard
+try:
+    import errors
+except BaseException:
+    from . import errors
+
+# create Artifacts folder paths
+base_folder_name = shipyard.logs.determine_base_artifact_folder(
+    'mode')
+artifact_subfolder_paths = shipyard.logs.determine_artifact_subfolders(
+    base_folder_name)
+shipyard.logs.create_artifacts_folders(artifact_subfolder_paths)
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--account-id', dest='account_id', required=True)
```

Comment:
Same note as the other PR. switch from `id` to `name` for clarity

Reply:
I realized what the potential issue might be with this one. I had made the report-run-id optional thinking that someone would first run the trigger report and would save it to a pickle file, but during the standup with Steven we had a while back that's not necessarily the case so entering your report-run-id in the CLI should be **mandatory**

Without both the report ID **and** the run ID specified the report will instead return a missing id message.